apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/ErrorReporting/ErrorReportingExceptionLogger.cs

```diff
@@ -145,8 +145,8 @@ private HttpRequestContext CreateHttpRequestContext(HttpContext context)
             return new HttpRequestContext()
             {
                 Method = request?.Method?.ToString() ?? "",
-                Url = UriHelper.GetDisplayUrl(request) ?? "",
-                UserAgent = request?.Headers["User-Agent"] ?? "",
+                Url = request?.GetDisplayUrl() ?? "",
+                UserAgent = request?.Headers["User-Agent"].ToString() ?? "",
```

Comment:
Ah - they're not actually StringValue, they're string fields according to https://github.com/googleapis/googleapis/blob/master/google/devtools/clouderrorreporting/v1beta1/common.proto - that's why they'd throw. It's fine to set a StringValue field to null.